### PR TITLE
Refs 184: add ouia and tooltips

### PR DESCRIPTION
**Pull request by swadeley**
Hello

Further to 
https://github.com/content-services/content-sources-frontend/pull/47

Adding some more OUIA IDs for the edit form and fixing surrounding tool tips.

https://issues.redhat.com/browse/HMSCONTENT-184

### CODE REVIEW

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMSCONTENT-184

**Comment by jlsherrill** [comment]
⚠️ This task currently requires qe-approval, but this PR does not fully resolve the task.  Please contact QE to determine appropriate testing required.

File: src/components/EditContentModal/EditContentModal.tsx [hunk 1]
```diff
@@ -364,8 +364,9 @@ const EditContentModal = ({ values, open, setClosed }: EditContentProps) => {
                     </FormGroup>
                     <FormGroup
                       label='Restrict architecture'
+                      ouiaId = 'restrict_architecture'
                       labelIcon={
-                        <Tooltip content='Something super important and stuff'>
+                        <Tooltip content='Optional: Select value to restrict package architecture'>
```
Comment (Andrewgdewar) [inline src/components/EditContentModal/EditContentModal.tsx:369]: These sound better to me, but you'll need to add the same thing to the "AddContentModal" as well.

Comment (swadeley) [reply src/components/EditContentModal/EditContentModal.tsx:369]: Done

Comment (swadeley) [reply src/components/EditContentModal/EditContentModal.tsx:383]: I see build errors, please review them

File: src/components/EditContentModal/EditContentModal.tsx [hunk 2]
```diff
@@ -252,7 +252,7 @@ const EditContentModal = ({ values, open, setClosed }: EditContentProps) => {
       }
       description={
         <p className={classes.description}>
-          Edit by completing the form. Default values may be provided automatically.
```
Comment (Andrewgdewar) [inline src/components/EditContentModal/EditContentModal.tsx:255]: Any changes to existing text will have to be changed in the corresponding test.

Comment (Andrewgdewar) [reply src/components/EditContentModal/EditContentModal.tsx:255]: I'll push an update

File: src/components/EditContentModal/EditContentModal.tsx (hunk 1 above)
Comment (Andrewgdewar) [reply src/components/EditContentModal/EditContentModal.tsx:383]: Reviewing. 
I may push updated changes to get this across the finish line.

File: src/components/EditContentModal/EditContentModal.tsx [hunk 3]
```diff
@@ -234,15 +234,11 @@ const EditContentModal = ({ values, open, setClosed }: EditContentProps) => {
     <Modal
       variant={ModalVariant.medium}
       title='Edit custom repository'
+      ouiaId='edit_custom_repository'
       help={
         <Popover
           headerContent={<div>Help Popover</div>}
```
Comment (swadeley) [inline src/components/EditContentModal/EditContentModal.tsx:240]:
```suggestion
          headerContent={<div>Edit custom repository</div>}
```

File: src/components/EditContentModal/EditContentModal.tsx [hunk 4]
```diff
@@ -234,15 +234,11 @@ const EditContentModal = ({ values, open, setClosed }: EditContentProps) => {
...
-          bodyContent={
-            <div>
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam id feugiat augue, nec
-              fringilla turpis.
-            </div>
-          }
+          bodyContent={<div>Use this form to edit the values of an existing repository.</div>}
           footerContent='Popover Footer'
```
Comment (swadeley) [inline src/components/EditContentModal/EditContentModal.tsx:242]:
```suggestion
          footerContent='* mandatory fields'
```

**Review by swadeley — COMMENTED** [review]
Some edits from pr #49

File: src/components/EditContentModal/EditContentModal.tsx (hunk 4 above)
Comment (Andrewgdewar) [reply src/components/EditContentModal/EditContentModal.tsx:242]: Capitalize first letter

File: src/components/AddContent/AddContent.tsx [hunk 5]
```diff
@@ -328,12 +328,7 @@ const AddContent = ({ isLoading }: Props) => {
...
             <Popover
               headerContent={<div>Help Popover</div>}
-              bodyContent={
-                <div>
-                  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam id feugiat augue,
-                  nec fringilla turpis.
-                </div>
-              }
+              bodyContent={<div>Use this form to enter the values for a new repository.</div>}
               footerContent='Popover Footer'
```
[the remark this answers is not in the document]
Comment (swadeley) [reply src/components/AddContent/AddContent.tsx:332]: @avisiedo  agreed we can just remove this

File: src/components/EditContentModal/EditContentModal.tsx (hunk 4 above)
Comment (swadeley) [reply src/components/EditContentModal/EditContentModal.tsx:242]: @avisiedo agreed we can just remove this

File: src/components/AddContent/AddContent.tsx [hunk 6]
```diff
@@ -327,14 +327,9 @@ const AddContent = ({ isLoading }: Props) => {
...
-              bodyContent={
-                <div>
-                  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam id feugiat augue,
-                  nec fringilla turpis.
-                </div>
-              }
-              footerContent='Popover Footer'
+              headerContent={<div>Add a custom repository</div>}
+              bodyContent={<div>Use this form to enter the values for a new repository.</div>}
+
```
Comment (Andrewgdewar) [inline src/components/AddContent/AddContent.tsx:332]: there is an empty space here that irks me.. kill it with fire!

Comment (Andrewgdewar) [inline src/components/AddContent/AddContent.tsx:332]:
```suggestion
```

Comment (swadeley) [reply src/components/AddContent/AddContent.tsx]: Done, thanks

File: src/components/EditContentModal/EditContentModal.tsx [hunk 7]
```diff
@@ -234,15 +234,11 @@ const EditContentModal = ({ values, open, setClosed }: EditContentProps) => {
...
-          headerContent={<div>Help Popover</div>}
-          bodyContent={
-            <div>
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam id feugiat augue, nec
-              fringilla turpis.
-            </div>
-          }
+          headerContent={<div>Edit custom repository</div>}
+          bodyContent={<div>Use this form to edit the values of an existing repository.</div>}
           footerContent='Popover Footer'
```
Comment (Andrewgdewar) [inline src/components/EditContentModal/EditContentModal.tsx:242]:
```suggestion
```

**Review by Andrewgdewar — APPROVED** [review]
Acked, feel free to merge when confident of changes

**Review by avisiedo — APPROVED** [review]
lgtm

**Pull request merged**